browser/extensions/api/brave_action_api.h

```diff
@@ -29,6 +29,14 @@ class BraveActionAPI : public KeyedService {
     virtual ~Observer();
   };
 
+  class ActionUIObserver {
```

Comment:
why does this need a new observer?

Reply:
It was a way to observe rewards popup close. Was discussed here:
https://github.com/brave/brave-core/pull/9611#discussion_r694980088

Reply:
I'm asking why this needs a new observer class. It already has one, why wouldn't we just add the method there?

Reply:
also I would change the name to `OnPopupClosed` for consistency

Reply:
code was removed as we catching popup close using js extension function call.